catalog/imaging/imaging-postprocess-plugin-nitf/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="Nitf Post Processing Plugin"
+         id="NITF_PostProcess_Plugin">
+        <AD
+                description="Maximum length of longest side of NITF overview image in pixels. The input transformer will calculate the size of the shorter side so that the overview will have the same aspect ratio as the original."
```

Comment:
❓Is this accurate: `will calculate the size of the shorter side`? Shouldn't the transformer use the larger of the two and use that to compute the proper sizing factor to keep the ratio?
Should we just say something like `will resize the image as needed so that the overview retains the same aspect ratio`?